org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java

```diff
@@ -635,6 +639,38 @@ public Tab insertAfter(Tab refTab, Component comp, String title, boolean closeab
     		return insertBefore((Tab)refTab.getNextSibling(), comp, title, closeable, enable, decorateInfo);
     }
 
+    /**
+     * IDEMPIERE-5275 - Tabular Report Re-Run button/close parameter window
+     * @param refTab 
+     * @param comp
+     * @param title
+     * @return
+     */
+    public Tab replace(Tab refTab, Window comp, String title) {
+    	 
+         if (refTab == null)  
+         {
+         	tabbox.getTabs().getChildren();
```

Comment:
what is this for ? Since refTab must not be null here, probably should just throw IllegalArgumentException here.